helm/istio/charts/tracing/templates/jaeger-all-in-one.yaml

```diff
@@ -0,0 +1,78 @@
+{{ if and (eq .Values.provider "jaeger") (eq .Values.jaeger.template "all-in-one") }}
+apiVersion: jaegertracing.io/v1
+kind: "Jaeger"
+metadata:
+  name: "jaeger"
+  namespace: {{ .Release.Namespace }}
+  labels:
+    chart: {{ template "tracing.chart" . }}
+    heritage: {{ .Release.Service }}
+    release: {{ .Release.Name }}
+spec:
+  strategy: allInOne
+
+  allInOne:
+    image: {{ .Values.jaeger.hub }}/all-in-one:{{ .Values.jaeger.tag }}
+    options:
+      log-level: debug
+    annotations:
+      {{- range $key, $value := .Values.jaeger.annotations }}
+        {{ $key }}: {{ $value | quote }}
+      {{- end }}
+
+  query:
+    options:
+      query:
+        base-path: /jaeger
+        ui-config: /conf/ui.json
+
+  collector:
```

Comment:
This `collector` and sub-nodes can be removed - the zipkin port is the default anyway so does not need to be specified.